### ISSUE TITLE
Hyperlinks not visible

### ISSUE THREAD

**Issue by stellargo** [coala/coala#4648]
The hyperlinks of the notes boxes at "http://api.coala.io/en/latest/Developers/Newcomers_Guide.html" have very low visibility. It is hard to distinguish from regular text. Although this doesn't seem to be a browser related, still the screenshot was taken on "Google Chrome Version 59.0.3071.115 (Official Build) (64-bit)".
![coala](https://user-images.githubusercontent.com/27735438/29489893-ef80fef2-8548-11e7-86ae-18e46a430d24.png)

**Comment by SanketDG** [comment]
This is how it appears in mine

![screenshot from 2017-08-20 14-38-52](https://user-images.githubusercontent.com/8980971/29493509-79503e2e-85b5-11e7-99a9-85ba22dcd5da.png)

Version 60.0.3112.90 (Official Build) (64-bit)

**Comment by Makman2** [comment]
Looks the same for me like @SanketDG 's example.

**Comment by prashant0598** [comment]
![capture](https://user-images.githubusercontent.com/24618078/29621435-534e8466-883e-11e7-98c4-374943df0cf7.PNG)
It looks perfect in Google Chrome Version 60.0.3112.101 (Official Build) (64-bit)

**Comment by Makman2** [comment]
Though interesting, it seems to look slightly different between Chrome and Firefox. Maybe it's really something internal of those browsers regarding text rendering (e.g. anti-aliasing maybe)? Version 60.xxx of @prashant0598 looks for me a bit better than your 59.xxx @stellargo.

**Comment by Makman2** [comment]
However we should maybe consider changing the style of links anyway.

**Comment by Makman2** [comment]
@stellargo do you have maybe Firefox installed? Then you could test that this might be a browser issue :+1:

**Comment by stellargo** [comment]
@Makman2 
Here is a screenshot taken on Safari Version 10.0 (11602.1.50.0.10)
![safari_coala](https://user-images.githubusercontent.com/27735438/29627444-bb6c3988-884f-11e7-99d7-3a6637babc04.png)
And here is one taken on Firefox 55.0.2 (64-bit)
![firefox_coala](https://user-images.githubusercontent.com/27735438/29627567-17c35aa4-8850-11e7-91ad-06ba39711a56.png)

**Comment by stellargo** [comment]
@Makman2 due to minor differences even I believe it could be some internal browser issue.
Just for the sake of clarity changing style of links would be a good idea.

**Comment by Makman2** [comment]
I would suggest using a dotted-underline for links, like here:

![](https://i.stack.imgur.com/W9WPi.png)

Copied from [stackoverflow](https://stackoverflow.com/questions/40777825/extra-dot-for-css-dotted-underline).

@stellargo you think that would fit? :)

**Comment by stellargo** [comment]
Yeah, I mean the one already present is very inconspicuous. Your example seems to have dots with more distinction

**Comment by Makman2** [comment]
Oh indeed I haven't seen the existing ones at all :O So yeah let's make them bigger :+1:

**Comment by prashant0598** [comment]
@Makman2 so how should I do it

**Comment by Makman2** [comment]
Dig somewhere in our layout files for documentation that defines the style for hyprerlinks. I don't know the exact location^^

**Comment by prashant0598** [comment]
@Makman2 @stellargo  Can you please direct me to the location wherever the change is required? Bcoz I could not find it in the repo.

**Comment by stellargo** [comment]
@Makman2 @prashant0598 Well my guess is the docs do contain the .rst file for the getting started page (Newcomers_Guide.rst) but what is being viewed on the website(CSS and HTML?) is not there in the repository. I may be wrong though.

**Comment by nishgaba-ai** [comment]
I found a fix i think, the only problem in visibility is the issue in the boxes, the file to change it is /docs/_static/custom.css

If we inspect the elements, the problem is in the visibility due to the color of underline too, if we just make the color of the underline (text-decoration) to be same color as the link's text(#eee) then the visibility is clear and easily recognizable.
I am going add this property in it.
![selection_004](https://user-images.githubusercontent.com/30121375/32996058-ab2e9982-cda3-11e7-935b-bdac3898a143.png)

**Comment by SanketDG** [comment]
@nishgaba-ai Looks like the theme we use does not use the underline property of css for underlining, instead a `border-bottom` with some padding. So, using the `text-decoration` property would mess up the hover style.

Instead something like:

```
border-bottom: 1px dashed #eee;
padding-bottom: 1px;
```

should look much better.

**Comment by jayvdb** [comment]
We are in the process of migrating & merging our various Sphinx config & style files.
